testing/kuttl/e2e/major-upgrade/01--postgres-connect.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
💭 but also ⛏️  and :thinking_dinosaur: This may be a trivial issue, but in my conception, we have two types of files: files to arrange the situation (create this cluster, update this cluster), and files to test this situation (make sure the cluster exists, make sure the cluster doesn't exist). 

To me, a script that checks the version is a test file, so should be `01-postgres-connect` rather than `01--postgres-connect`. What do you think?

Reply:
I went back and forth a bit about this, and I think I could go either way. I'll go ahead and remove the extra `-` since I noticed that's where we've  already landed.